### ISSUE TITLE
CPU 143% use

### ISSUE THREAD

**Issue by krafg** [SilverAzide/Gadgets#111]
My Ryzen 7 4800H have more than 100% of use. I can not understand.
https://i.imgur.com/VxexQZr.png

Regards.

**Comment by SilverAzide** [comment]
This is normal and expected.  The base clock of your 4800H CPU is 2.9 GHz, or 2900 Mhz.  Your CPU in your screenshot is running in turbo boost mode at 4140 MHz.  If you do the math,  `4140 ÷ 2900 = 1.4275`, which is 143%.  This matches exactly with the overall percentage usage shown in the skin.

Due to modern CPUs that overclock and underclock automatically, plus hybrid CPUs with efficiency cores that run at different clock speeds than the performance cores, Microsoft has long since stopped reporting CPU usages percentages that peak at 100%.  If you want more information, read this wiki article here:

**[INFO: CPU performance exceeds 100%](https://github.com/SilverAzide/Gadgets/wiki/INFO-CPU-performance-exceeds-100)**

The TLDR; version:  time-based performance counters are no longer good indicators of CPU utilization, so Microsoft today uses a different method that take into account clock speed.

As mentioned in the wiki article, if you would prefer using the old-style Windows 7 performance counters, open the CPU Settings and select the **Use Legacy Mode** option.

![image](https://github.com/user-attachments/assets/a2985c9a-5f67-4ad5-adb2-070ab1094f1e)

**Comment by krafg** [comment]
Thanks a lot.
